tests/python/gaia-ui-tests/gaiatest/tests/graphics/RTL/test_settings_NC_RTL.py

```diff
@@ -0,0 +1,62 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
okay, turns out it was a timing issue, i ran it multiple times and this time it definitely passed.